src/Lean/Server/Watchdog.lean

```diff
@@ -310,9 +311,9 @@ section ServerM
 
   /-- Creates a Task which forwards a worker's messages into the output stream until an event
   which must be handled in the main watchdog thread (e.g. an I/O error) happens. -/
-  private partial def forwardMessages (fw : FileWorker) : ServerM (Task WorkerEvent) := do
-    let task ← IO.asTask (loop $ ←read) Task.Priority.dedicated
-    return task.map fun
+  private partial def forwardMessages (fw : FileWorker) : ServerM (ServerTask WorkerEvent) := do
+    let task ← ServerTask.IO.asTask (loop $ ←read)
```

Comment:
It looks like `ServerTask.IO.asTask` directly calls `IO.asTask`, but the default priority of that is `default`, so this is no longer `dedicated`?

Reply:
Argh, great catch. Thanks!